### ISSUE TITLE
Safe to add "extends HiveObject" later?

### ISSUE THREAD

**Issue by MLZ91** [hivedb/hive#1253]
I have an older project with users and a Hive database. I realized that I forgot to add "extends HiveObject" to one object when I wanted to access the key.

Is it safe to add "extends HiveObject" now?

**Comment by hamza-ezzahiry** [comment]
Yes, it is generally safe to add `extends HiveObject` to your class in Hive, even after you have already stored data in the database.
